### ISSUE TITLE
[Request] Ability to change chainlift speed or a booster track

### ISSUE THREAD

**Issue by ShoobFloof22** [ImplicitSaber/Forkcart#9]
Basically what the title says, I think it would be nice to have the option to change the chainlift speed or have a magnetic launching track of some kind. The original Splinecart's chainlift was really fast, and I know some chainlifts usually aren't, but they also had a magnetic launch track that was basically a speed booster. It would be nice to have a booster track in this fork, or at least to change the chainlift speed, because when making huge coasters, the slow chainlift speed takes EONS to get you up to the top lol. 
For what it's worth I realize you could just use the original mod if you wanted the faster lifts, but at least in my case, I was entirely unable to get carts to go back to normal tracks. They'd just stop dead at the transition point between splines and normal track, and without a station or switch track, you basically just sent the cart on its way and that was that, no stopping it unless you broke it. This mod fixes both those issues which I greatly appreicate. I just wish we had faster options for bring carts up high lol

**Comment by Arcdashckr** [comment]
I totally agree with what is written above also it would be nice if we could use it with the rails used by the splinecart, and also the sound made by the chainlift can be disturbing, it would be better if it was quiet
